types/get-installed-path/index.d.ts

```diff
@@ -0,0 +1,13 @@
+// Type definitions for get-installed-path 4.0
+// Project: https://github.com/tunnckoCore/get-installed-path
+// Definitions by: Jamie Magee <https://github.com/JamieMagee>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export function getInstalledPath(name: string, opts?: Options): Promise<string>;
+export function getInstalledPathSync(name: string, opts?: Options): string;
+
+export interface Options {
+    cwd?: string;
+    local?: boolean;
+    paths?: string[];
```

Comment:
In [the source code](https://github.com/tunnckoCore/get-installed-path/blob/520646179ef9ac00a8d8cb8c54afb1f5e7dd798f/src/index.js#L164), this array does not change. I propose to replace it with:

```suggestion
    paths?: readonly string[];
```